src/docs/user.yaml

```diff
@@ -28,3 +28,27 @@ paths:
           description: 'Unauthorized (one needs to be logged in)'
         '500':
           description: 'Internal server error (ID is not a valid ObjectId or a server error occurred)'
+    delete:
+      tags:
+        - User
```

Comment:
we can add summary after this line something like "Deletes profile of the user with given ID"